### PR TITLE
feat(flux): Se desvincula la libería yOSON

### PR DESCRIPTION
**Pull request by carloshs92**
- Cambios en la estructura de carpetas en pre_css, pre_js y pre_html
- Cambios en el renderizado de secciones del servidor expressjs
- Se retira la dependencia que tiene flux con el yoson para la realizacion
de las pruebas con mocha
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/jansanchez%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2623671%3Fv%3D3%22%7D%2C%20%22https%3A//github.com/andru255%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/729789%3Fv%3D3%22%7D%2C%20%22https%3A//github.com/lavaldi%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1221474%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/frontend-labs/flux/pull/45%23issuecomment-128764281%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-08-07T16%3A55%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1221474%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/lavaldi%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40jansanchez%20%20Revisa%20por%20favor%20%3Abook%3A%20%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222015-08-07T16%3A58%3A16Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1372633%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/carloshs92%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-08-07T17%3A16%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1372633%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/carloshs92%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-08-07T17%3A22%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2623671%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jansanchez%22%7D%7D%2C%20%7B%22body%22%3A%20%22Info%3A%20Este%20pull%20request%20genero%204%20commits.%22%2C%20%22created_at%22%3A%20%222015-08-07T17%3A22%3A34Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2623671%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jansanchez%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-08-07T17%3A30%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/729789%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/andru255%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222015-08-07T17%3A34%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/729789%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/andru255%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20b4feff6827a79e89e046417a1011439ad4d1782c%20frontend/config/tasks/gulp_html.js%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/frontend-labs/flux/pull/45%23discussion_r36537151%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Aqu%5Cu00ed%20hay%202%20lineas%20con%20indentaci%5Cu00f3n%20fuera%20de%20lo%20com%5Cu00fan.%22%2C%20%22created_at%22%3A%20%222015-08-07T16%3A46%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2623671%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jansanchez%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-08-07T16%3A57%3A50Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2623671%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jansanchez%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/config/tasks/gulp_html.js%3AL33-40%22%7D%2C%20%22Pull%20b4feff6827a79e89e046417a1011439ad4d1782c%20frontend/source/pre_js/sections/home/inicio.coffee%202%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/frontend-labs/flux/pull/45%23discussion_r36537093%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Lo%20ideal%20seria%20no%20usar%20%5C%22SpanGlish%5C%22%2C%20menos%20si%20es%20un%20ejemplo.%5Cr%5CnAdicionalmente%20corregir%20%5C%22lenguage%5C%22.%22%2C%20%22created_at%22%3A%20%222015-08-07T16%3A46%3A18Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2623671%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jansanchez%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-08-07T16%3A57%3A50Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2623671%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jansanchez%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/source/pre_js/sections/home/inicio.coffee%3AL1-9%22%7D%2C%20%22Pull%20b4feff6827a79e89e046417a1011439ad4d1782c%20frontend/tests/mocha/sections/home/test_default.js%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/frontend-labs/flux/pull/45%23discussion_r36537063%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Ser%5Cu00eda%20ideal%20%2C%20no%20usar%20%5C%22Spanglish%5C%22.%5Cr%5Cn%5Cr%5CnSe%20sugiere%20que%20la%20funci%5Cu00f3n%20%5C%22saludar%5C%22%20se%20renombre%20%5C%22sayHi%5C%22%20ya%20que%20el%20c%5Cu00f3digo%20de%20ejemplo%20esta%20en%20ingl%5Cu00e9s.%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222015-08-07T16%3A45%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2623671%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jansanchez%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222015-08-07T16%3A57%3A50Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/2623671%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/jansanchez%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20frontend/tests/mocha/sections/home/test_default.js%3AL1-16%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/frontend-labs/flux/pull/45%23issuecomment-128764281%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23issuecomment-128765453%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23issuecomment-128770296%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23issuecomment-128771493%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23issuecomment-128771586%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23issuecomment-128773094%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23issuecomment-128773841%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23discussion_r36537063%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23discussion_r36537093%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23discussion_r36537151%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23discussion_r36538083%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23discussion_r36538085%22%2C%20%22https%3A//github.com/frontend-labs/flux/pull/45%23discussion_r36538086%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/jansanchez'></a><a href='https://github.com/andru255'></a><a href='https://github.com/lavaldi'></a>

- [x] <a href='#crh-comment-Pull b4feff6827a79e89e046417a1011439ad4d1782c frontend/tests/mocha/sections/home/test_default.js 3'></a> &nbsp;<b><a href='https://github.com/frontend-labs/flux/pull/45#discussion_r36537063'>File: frontend/tests/mocha/sections/home/test_default.js:L1-16</a></b>
- <a href='https://github.com/jansanchez'></a> Sería ideal , no usar "Spanglish".
Se sugiere que la función "saludar" se renombre "sayHi" ya que el código de ejemplo esta en inglés.
- [x] <a href='#crh-comment-Pull b4feff6827a79e89e046417a1011439ad4d1782c frontend/source/pre_js/sections/home/inicio.coffee 2'></a> &nbsp;<b><a href='https://github.com/frontend-labs/flux/pull/45#discussion_r36537093'>File: frontend/source/pre_js/sections/home/inicio.coffee:L1-9</a></b>
- <a href='https://github.com/jansanchez'></a> Lo ideal seria no usar "SpanGlish", menos si es un ejemplo.
Adicionalmente corregir "lenguage".
- [x] <a href='#crh-comment-Pull b4feff6827a79e89e046417a1011439ad4d1782c frontend/config/tasks/gulp_html.js 3'></a> &nbsp;<b><a href='https://github.com/frontend-labs/flux/pull/45#discussion_r36537151'>File: frontend/config/tasks/gulp_html.js:L33-40</a></b>
- <a href='https://github.com/jansanchez'></a> Aquí hay 2 lineas con indentación fuera de lo común.
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/frontend-labs/flux/pull/45#issuecomment-128764281'>General Comment</a></b>
- <a href='https://github.com/lavaldi'></a> :shipit:
- <a href='https://github.com/carloshs92'></a> @jansanchez  Revisa por favor :book:
- <a href='https://github.com/carloshs92'></a> :+1:
- <a href='https://github.com/jansanchez'></a> :shipit:
- <a href='https://github.com/jansanchez'></a> Info: Este pull request genero 4 commits.

<a href='https://www.codereviewhub.com/frontend-labs/flux/pull/45?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/frontend-labs/flux/pull/45?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/frontend-labs/flux/pull/45?approve=0'></a>&nbsp;<a href='https://github.com/frontend-labs/flux/pull/45'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by andru255** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/frontend-labs/flux/pull/45'>Refresh</a>.<a href='#crh-update-none'></a>

File: frontend/tests/mocha/sections/home/test_default.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+describe('test_default', function(){
+
+	describe("#saludar()", function () {
```
Comment (jansanchez) [inline frontend/tests/mocha/sections/home/test_default.js]: Sería ideal , no usar "Spanglish".

Se sugiere que la función "saludar" se renombre "sayHi" ya que el código de ejemplo esta en inglés.

File: frontend/source/pre_js/sections/home/inicio.coffee [hunk 2]
```diff
@@ -0,0 +1,8 @@
+fn =
+	saludar: (lenguage)->
```
Comment (jansanchez) [inline frontend/source/pre_js/sections/home/inicio.coffee]: Lo ideal seria no usar "SpanGlish", menos si es un ejemplo.
Adicionalmente corregir "lenguage".

File: frontend/config/tasks/gulp_html.js [hunk 3]
```diff
@@ -33,9 +33,7 @@ function Task(gulp, path, config, plugins, functions){
 					staticUrl : "/",
 					elementUrl: "/",
 					version 	: new Date().getTime(),
```
Comment (jansanchez) [inline frontend/config/tasks/gulp_html.js]: Aquí hay 2 lineas con indentación fuera de lo común.

**Comment by lavaldi** [comment]
:shipit:

File: frontend/config/tasks/gulp_html.js (hunk 3 above)
Comment (jansanchez) [inline frontend/config/tasks/gulp_html.js]: :+1:<a href='#crh-update-none'></a>

File: frontend/source/pre_js/sections/home/inicio.coffee (hunk 2 above)
Comment (jansanchez) [inline frontend/source/pre_js/sections/home/inicio.coffee]: :+1:<a href='#crh-update-none'></a>

File: frontend/tests/mocha/sections/home/test_default.js (hunk 1 above)
Comment (jansanchez) [inline frontend/tests/mocha/sections/home/test_default.js]: :+1:<a href='#crh-update-complete'></a>

**Comment by carloshs92** [comment]
@jansanchez  Revisa por favor :book:

**Comment by carloshs92** [comment]
:+1:

**Comment by jansanchez** [comment]
:shipit:

**Comment by jansanchez** [comment]
Info: Este pull request genero 4 commits.

**Comment by andru255** [comment]
:+1:

**Comment by andru255** [comment]
:shipit:

**Pull request merged**